pkg/exposer/generic_restore.go

```diff
@@ -143,7 +144,20 @@ func (e *genericRestoreExposer) GetExposed(ctx context.Context, ownerObject core
 
 	curLog.WithField("restore pvc", restorePVCName).Info("Restore PVC is bound")
 
-	return &ExposeResult{ByPod: ExposeByPod{HostingPod: pod, VolumeName: pod.Spec.Volumes[0].Name}}, nil
+	i := 0
+	for i = 0; i < len(pod.Spec.Volumes); i++ {
+		if pod.Spec.Volumes[i].Name == volumeName {
+			break
+		}
+	}
+
+	if i == len(pod.Spec.Volumes) {
+		return nil, errors.Errorf("restore pod %s doesn't have the expected restore volume", pod.Name)
+	}
+
+	curLog.WithField("pod", pod.Name).Infof("Restore volume is found in pod at index %v", i)
```

Comment:
Unsure if this log should be info level ?

Reply:
This can be info log, it is printed only once for each volume and it could help us on the troubleshooting.